lib/screens/v2/widget_instance/alert.ex

```diff
@@ -32,7 +32,7 @@ defmodule Screens.V2.WidgetInstance.Alert do
   @automated_override_priority [1, 2]
 
   # Keep these in descending order of priority--highest priority (lowest integer value) first
-  @relevant_effects ~w[shuttle stop_closure suspension station_closure detour stop_moved snow_route elevator_closure]a
+  @relevant_effects ~w[shuttle stop_closure suspension station_closure detour service_change stop_moved snow_route elevator_closure]a
```

Comment:
Are the changes in this module relevant to the Subway Status widget? Although maybe we _could_ support service change alerts in this other widget, I don't think these changes alone are sufficient to do that.